src/components/Menu/Menu.tsx

```diff
@@ -5,15 +5,20 @@ import { Platform } from '@/data/platforms';
 import Link from 'next/link';
 import { LayoutContext } from '@/components/Layout';
 import { PageNode } from 'src/directory/directory';
+import { Url } from 'next/dist/shared/lib/router/router';
```

Comment:
Is this import really needed?  Seems to me like this is the kind of path that will disappear on us in the future 🤷‍♂️

Reply:
Ah I see what you mean, my bad. I'll see if I can find another import or maybe make my own type. I'm using that as the type for `MenuProps` basically

Reply:
@jacoblogan, just pushed an update to use `LinkProps` from `next/link` and I reference the `URL` type by doing `LinkProps['href']`